src/kvirc/kernel/KviOptions.cpp

```diff
@@ -874,17 +874,17 @@ KviMessageTypeSettingsOption g_msgtypeOptionsTable[KVI_NUM_MSGTYPE_OPTIONS]=
 	MSGTYPE_OPTION("MeChanUnOwner",__tr_no_lookup("Own chan owner status unset"),KviIconManager::MeChanUnOwner,KVI_MSGTYPE_LEVEL_5),
 	MSGTYPE_OPTION_SPEC("ServerError",__tr_no_lookup("Server error"),KviIconManager::ServerError,KviControlCodes::Red,KviControlCodes::Transparent,KVI_MSGTYPE_LEVEL_5),
 	MSGTYPE_OPTION_SPEC("Help",__tr_no_lookup("Help"),KviIconManager::Help,KviControlCodes::DarkGreen,KviControlCodes::Transparent,KVI_MSGTYPE_LEVEL_1),
-	MSGTYPE_OPTION("TextEncoding",__tr_no_lookup("Text Encoding Server message"),KviIconManager::TextEncoding,KVI_MSGTYPE_LEVEL_1),
+	MSGTYPE_OPTION("TextEncoding",__tr_no_lookup("Text encoding server message"),KviIconManager::TextEncoding,KVI_MSGTYPE_LEVEL_1),
 	// TODO: add icon and check what level means
 	// Pragma: the message level is a warning level that is rendered in the WindowList
 	//         messages with higher level are rendered with more visible color (i.e. red)
 	//         messages with very high level might flash the tray and end up in the notifier
 	//         level 1 or 2 for standard torrent messages is ok
 	MSGTYPE_OPTION("BitTorrent",__tr_no_lookup("BitTorrent message"),KviIconManager::Gnutella,KVI_MSGTYPE_LEVEL_2),
-	MSGTYPE_OPTION("IrcOp",__tr_no_lookup("IRC Op status set"),KviIconManager::IrcOp,KVI_MSGTYPE_LEVEL_5),
-	MSGTYPE_OPTION("DeIrcOp",__tr_no_lookup("IRC Op status unset"),KviIconManager::DeIrcOp,KVI_MSGTYPE_LEVEL_5),
-	MSGTYPE_OPTION("MeIrcOp",__tr_no_lookup("Own IRC Op status set"),KviIconManager::MeIrcOp,KVI_MSGTYPE_LEVEL_5),
-	MSGTYPE_OPTION("MeDeIrcOp",__tr_no_lookup("Own IRC Op status unset"),KviIconManager::MeDeIrcOp,KVI_MSGTYPE_LEVEL_5),
+	MSGTYPE_OPTION("IrcOp",__tr_no_lookup("IRC op status set"),KviIconManager::IrcOp,KVI_MSGTYPE_LEVEL_5),
```

Comment:
These should probably just be `IRC Op` or `IRC Operator` (The later is the formal way of saying it). In any event, keep `Op` or `Operator` capitalized since it's a proper noun in this sense.